src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +11,10 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
+        UserService user = new UserService();
+        if (user.findByEmail(email) != null) {
+            return user.findByEmail(email).getPassword().equals(password);
+        }
         return false;
```

Comment:
```suggestion
            return user != null && user.getPassword().equals(password);
        }
```
don't call findByEmail twice, store the result of the call in a local variable, also, we can combine expressions like this

Reply:
Condition 'user != null' is always 'true'